src/util/mocks.tsx

```diff
@@ -89,10 +124,64 @@ export const chatWithReads = {
     },
   ],
   last_message: message,
-  sender: adam,
 };
 
-export const chats = [
+export const threePersonChat: ChatProps = {
+  id: 207,
+  title: 'Friends!',
+  is_direct_chat: false,
+  created: '2021-01-28T02:41:48.826706Z',
+  custom_json: {},
+  attachments: [],
+  people: [
+    {
+      last_read: 1000,
+      person: adam,
+      chat_updated: '',
+    },
+    {
+      last_read: 1000,
+      person: bob,
+      chat_updated: '',
+    },
+    {
+      last_read: 1000,
+      person: cam,
+      chat_updated: '',
+    },
+  ],
+  last_message: message,
+};
+
+export const directMessageChat: ChatProps = {
+  id: 203,
+  title: 'Direct Message w/ Adam',
+  is_direct_chat: true,
+  created: '2021-01-26T02:41:48.826706Z',
+  custom_json: {},
+  attachments: [],
+  people: [
+    {
+      last_read: 1000,
+      person: adam,
+      chat_updated: '',
+    },
+    {
+      last_read: 1000,
+      person: bob,
+      chat_updated: '',
+    }
+  ],
+  last_message: {
+    created: '',
+    attachments: [],
+    sender_username: '',
+    text: 'hello',
+    custom_json: '',
+  },
+};
+
+export const chats: ChatProps[] = [
   {
     id: 201,
     title: 'First Chat ☝️',
```

Comment:
Also another thing to discuss: where should the mocks live? I feel like pulling in what's necessary to the component folder might be better practice. 

More repetition but mock's are scoped better.

Again, I don't know the answer we should chat about this one

Reply:
in the repos I worked on at work we followed a similar pattern to this where we had a static class called `TestObjects` and we would use it as a utility for pulling in test objects like this. Since this was a java project it lived in `src/test/`, but still away from the regular code

I'm ok leaving as is for now since it leaves the component folder less cluttered, but I'll have to look into what JS/TS projects usually do for this kinda thing

Reply:
Sounds good